src/Anonymization/Anonymizer/Core/FloatAnonymizer.php

```diff
@@ -23,27 +27,109 @@ class FloatAnonymizer extends AbstractAnonymizer
      */
     public function anonymize(Update $update): void
     {
-        if (!($this->options->has('min') && $this->options->has('max'))) {
-            throw new \InvalidArgumentException("You should provide 2 options (min and max) with this anonymizer");
+
+        $precision = $this->options->get('precision', 2);
+        if ($precision <= 0) {
+            throw new \InvalidArgumentException("'precision' should be greater than 0.");
         }
+        $precision = 10 ** $precision;
+
+        if ($this->options->has('min') && $this->options->has('max')) {
+            $min = (float) $this->options->get('min');
+            $max = (float) $this->options->get('max');
+            if ($min >= $max) {
+                throw new \InvalidArgumentException("'max' should be greater than 'min'.");
```

Comment:
Honestly, I would simply inverse both instead of raising an exception, and trigger a warning instead ?

Reply:
I never know what to do in such case.

In one hand, in most cases, you probably right, just inverse both could be a good solution. 
But what if it's a typo and `max` has a typo for example 100 instead of 1000 ? The error will pass silently :/

At least here, we warn :)

Reply:
I let you decide, whatever you choose it's OK for me. Triggering a warning in a logger is a good idea (we should inject a logger into anonymizers as well I guess).

Reply:
Anonymizers are not services, it won't be so straight forward to inject logger in it. Let's keep it like this for now.